web/src/engine/dom-utils/src/stylesheets.ts

```diff
@@ -197,15 +228,15 @@ export class StylesheetManager {
    *
    * @param   {string}  href   path to stylesheet file
    */
-  linkExternalSheet(href: string): void {
+  linkExternalSheet(href: string, force?: boolean): HTMLStyleElement {
```

Comment:
There's a change to this function signature but the change doesn't seem to be used?

Reply:
It comes into play in the followup - #10507.  Admittedly, the two were part of a single batch of changes at first, with me splitting them into separate chunks later.

If need be, I can undo it here and re-implement in that PR, which is a direct follow-up.

Reply:
no that's fine, I see the changes there.